dev-system/dev/user.clj

```diff
@@ -29,6 +32,26 @@
           (fn [app-map]
             (u/map-values #(assoc-in % [:log :level] level) app-map))))
 
+(defn set-mode!
+  "Set the mode to use when starting/resetting. Valid options are:
+
```

Comment:
How are you guys on defensive programming?

I tend to stay away from that sort of thing myself, especially if writing code that devs will use in a REPL ...